h/views/accounts.py

```diff
@@ -167,12 +162,6 @@ def _redirect_if_logged_in(self):
     def _login_redirect(self):
         return self.request.params.get("next", _login_redirect_url(self.request))
 
-    def _login(self, user):
-        user.last_login_date = datetime.datetime.utcnow()  # noqa: DTZ003
-        self.request.registry.notify(LoginEvent(self.request, user))
-        headers = security.remember(self.request, user.userid)
-        return headers
```

Comment:
Moved this code into a helper in a separate file so that the OICD views can also call it.